packages/qwik/src/core/render/jsx/types/jsx-qwik-attributes.ts

```diff
@@ -170,10 +181,6 @@ export interface QwikEvents {
 
   'document:onVisibilityChange$'?: (event: Event, el: Element) => any;
   'document:onVisibilityChangeQrl'?: QRL<(event: Event, el: Element) => any>;
```

Comment:
We can remove too

Reply:
the Qrl variant

Reply:
Cool. Also, document:onLoad$ and document:onScroll$ should already be generated from the new types. Should I delete those too?